mods/default/furnace.lua

```diff
@@ -133,13 +133,14 @@ local function furnace_node_timer(pos, elapsed)
 		cooked, aftercooked = minetest.get_craft_result({method = "cooking", width = 1, items = srclist})
 		cookable = cooked.time ~= 0
 
+		local el = math.min(elapsed, fuel_totaltime - fuel_time)
```

Comment:
Since we are looking for what is going to happen first (next fuel or the cooked item), this logic seems to apply more:
```Lua
local el = math.min(elapsed, fuel_totaltime - fuel_time)
if cookable then
	el = math.min(el, cooked.time - src_time)
end
```

Reply:
I thought `src_time` was counted correctly as well, but I didn't think about the one one item per fuel. Fixing.